diff --git a/csl-mlz.rnc b/csl-mlz.rnc
index b59537e..49f0019 100644
--- a/csl-mlz.rnc
+++ b/csl-mlz.rnc
@@ -16,7 +16,7 @@ dc:description [
 
 include "csl-jurisdictions.rnc"
 include "csl-repository.rnc" {
-  version = attribute version { "1.1mlz1" }
+  # version = attribute version { "1.1mlz1" }
   
   ## Add "skip-words" attribute
   locale.style-options =
@@ -95,66 +95,6 @@ rendering-element.date =
     | rendering-element.group
     | rendering-element.alternative
     | rendering-element.alternative-text
-
-## Terms (adding email, instant-message, television-broadcast, radio-broadcast, and podcast)
-  terms =
-    terms.gender-assignable
-    | terms.gender-variants
-    | terms.locator
-    | 
-      ## Contributor roles
-      variables.names
-    | "editortranslator"
-    | 
-      ## Miscellaneous terms
-      "accessed"
-    | "ad"
-    | "and"
-    | "and others"
-    | "anonymous"
-    | "at"
-    | "available at"
-    | "bc"
-    | "by"
-    | "circa"
-    | "cited"
-    | "email"
-    | "et-al"
-    | "forthcoming"
-    | "from"
-    | "ibid"
-    | "in"
-    | "in press"
-    | "instant-message"
-    | "internet"
-    | "interview"
-    | "letter"
-    | "no date"
-    | "online"
-    | "podcast"
-    | "presented at"
-    | "radio-broadcast"
-    | "reference"
-    | "retrieved"
-    | "scale"
-    | "television-broadcast"
-    | "version"
-    | 
-      ## Punctuation
-      "open-quote"
-    | "close-quote"
-    | "open-inner-quote"
-    | "close-inner-quote"
-    | "page-range-delimiter"
-    | 
-      ## Seasons
-      "season-01"
-    | "season-02"
-    | "season-03"
-    | "season-04"
-    |
-      ## (legacy; remove in CSL 1.1)
-      category.field
   
   date.form =
     
@@ -187,6 +127,13 @@ rendering-element.date =
       citation.layout-locale*,
       citation.layout-default
     }
+  style.intext =
+    element cs:intext {
+      citation.options,
+      sort?,
+      citation.layout-locale*,
+      citation.layout-default
+    }
   style.bibliography =
     element cs:bibliography {
       bibliography.options,
@@ -198,22 +145,6 @@ rendering-element.date =
   ## Support for complex conditions
   choose.if = ( if-simple | if-complex )
   choose.else-if = ( else-if-simple | else-if-complex )
-
-  ## Allow cs:institution element as child element of cs:names, add
-  ## name-label-before, name-label-after
-  rendering-element.names =
-    element cs:names {
-      names.attributes,
-      attribute require-match { xsd:boolean }?,
-      (names.label-before?,
-       names.name?,
-       names.et-al?,
-       names.institution?,
-       names.label-after?),
-      names.substitute?
-    }
-
-
   
   ## Replace affixes by affixes-date on cs:date-part
   rendering-element.date.date-part.non-localized =
@@ -226,32 +157,32 @@ rendering-element.date =
   
   ## Allow "label-form" attribute to accompany "macro" on cs:text
   ## Allow "leading-noise-words" attribute to accompany "variable" on cs:text
-  text.attributes =
-    (
-     ## Select a macro.
-     attribute macro { xsd:NMTOKEN },
-     attribute label-form { term.form }?,
-     [a:defaultValue = "false" ] attribute label-capitalize-if-first { xsd:boolean }?)
-    | (
-       ## Select a term.
-       attribute term { terms },
-       [ a:defaultValue = "long" ] attribute form { term.form }?,
+  # text.attributes =
+  #   (
+  #    ## Select a macro.
+  #    attribute macro { xsd:NMTOKEN },
+  #    attribute label-form { term.form }?,
+  #    [a:defaultValue = "false" ] attribute label-capitalize-if-first { xsd:boolean }?)
+  #   | (
+  #      ## Select a term.
+  #      attribute term { terms },
+  #      [ a:defaultValue = "long" ] attribute form { term.form }?,
        
-       ## Specify term plurality: singular ("false") or plural ("true").
-       [ a:defaultValue = "false" ] attribute plural { xsd:boolean }?,
-
-       ## Force use of the style default locale for the term
-       use-default-locale)
-    | 
-      ## Specify verbatim text.
-      attribute value { text }
-    | (
-       ## Select a variable.
-       attribute variable { variables.strings },
-       ## Form of the variable
-       [ a:defaultValue = "long" ] attribute form { "short" | "long" }?,
-       ## What to do with leading noise words
-       attribute leading-noise-words { "drop" | "demote" }?)
+  #      ## Specify term plurality: singular ("false") or plural ("true").
+  #      [ a:defaultValue = "false" ] attribute plural { xsd:boolean }?,
+
+  #      ## Force use of the style default locale for the term
+  #      use-default-locale)
+  #   | 
+  #     ## Specify verbatim text.
+  #     attribute value { text }
+  #   | (
+  #      ## Select a variable.
+  #      attribute variable { variables.standard  },
+  #      ## Form of the variable
+  #      [ a:defaultValue = "long" ] attribute form { "short" | "long" }?,
+  #      ## What to do with leading noise words
+  #      attribute leading-noise-words { "drop" | "demote" }?)
        
   ## Allow "locator" variable on, add "label-form" attribute to cs:number
   number.attributes =
@@ -374,26 +305,6 @@ rendering-element.date =
     ## or at or below a max value
     attribute suppress-min { xsd:integer }?,
     attribute suppress-max { xsd:integer }?
-
-  ## Drop affixes
-  label.attributes-shared =
-    
-    ## Specify when the plural version of a term is selected.
-    [ a:defaultValue = "contextual" ]
-    attribute plural { "always" | "never" | "contextual" }?,
-    font-formatting,
-    strip-periods,
-    text-case
-  
-  ## Add back affixes (removed from label.attributes-shared)
-  label.attributes =
-    attribute variable { variables.numbers | "locator" | "page" },
-    [ a:defaultValue = "long" ]
-    attribute form { "long" | "short" | "symbol" | "static" }?,
-    [ a:defaultValue = "false" ]
-    attribute capitalize-if-first { xsd:boolean }?,
-    use-default-locale,
-    affixes
   
   ## Add "is-parallel", "quotes", "subgroup-delimiter",
   ## "subgroup-delimiter-precedes-last", attribute on cs:group
@@ -442,7 +353,11 @@ rendering-element.date =
     attribute disable-duplicate-year-suppression {
       list { jurisdictions+ }
     }?,
-    attribute track-containers { list { item-types+ } }?
+    attribute track-containers { list { item-types+ } }?,
+  
+    # https://citeproc-js.readthedocs.io/en/latest/csl-m/index.html#default-locale-sort-extension
+    attribute default-locale-sort { list { xsd:language+ } }?
+
 
 
   
@@ -586,11 +501,7 @@ rendering-element.date =
         xsd:integer
       }
 
-  affixes =
-    [ a:defaultValue = "" ]
-    attribute prefix { string-no-leading-space }?,
-    [ a:defaultValue = "" ]
-    attribute suffix { string-no-trailing-space }?
+
   
   ## Add "normal" value on text-case attribute
   text-case =
@@ -621,56 +532,6 @@ rendering-element.date =
         "normal"
     }?
   
-  ## standard variables
-  variables.strings =
-    "abstract"
-    | "annote"
-    | "archive"
-    | "archive_location"
-    | "archive-place"
-    | "citation-label"
-    | "citation-number"
-    | "collection-title"
-    | "container-title"
-    | "container-title-short"
-    | "country"
-    | "dimensions"
-    | "DOI"
-    | "event"
-    | "event-place"
-    | "first-reference-note-number"
-    | "first-container-reference-note-number"
-    | "genre"
-    | "hereinafter"
-    | "ISBN"
-    | "ISSN"
-    | "jurisdiction"
-    | "keyword"
-    | "locator"
-    | "medium"
-    | "note"
-    | "original-publisher"
-    | "original-publisher-place"
-    | "original-title"
-    | "PMID"
-    | "PMCID"
-    | "publisher"
-    | "publisher-place"
-    | "references"
-    | "reviewed-title"
-    | "scale"
-    | "section"
-    | "source"
-    | "status"
-    | "title"
-    | "title-short"
-    | "title-main"
-    | "title-sub"
-    | "URL"
-    | "version"
-    | "volume-title"
-    | "year-suffix"
-
   independent-style.style.info =
     element cs:info {
       info.title,
@@ -868,7 +729,7 @@ affixes-trailing-name-label =
 div {
   
   ## Add locator terms
-  terms.locator.testable |= "Chapter" | "Section" | "article" | "rule" | "title" | "number"
+  terms.locator.testable |= terms.locator | "Chapter" | "Section" | "article" | "rule" | "title" | "number"
       | "illustration" | "introduction" | "preamble" | "principle" | "publication" | "schedule"
       | "series," | "subdivision" | "subsection" | "supplement" | "table" | "title"
       | "subparagraph" | "amendment" | "annotation" | "appendix" | "bibliography" | "clause"
diff --git a/csl.rnc b/csl.rnc
index ac6f9c5..ced1481 100644
--- a/csl.rnc
+++ b/csl.rnc
@@ -66,6 +66,7 @@ div {
       (style.locale*
        & style.macro*
        & style.citation
+       & style.intext?
        & style.bibliography?)
     }
   
@@ -401,6 +402,11 @@ div {
     ## Use to describe the formatting of citations.
     element cs:citation { citation.options, sort?, citation.layout }
   
+  style.intext =
+    
+    ## Use to describe the formatting of citations.
+    element cs:intext { citation.options, sort?, citation.layout }
+  
   style.bibliography =
     
     ## Use to describe the formatting of the bibliography.
