diff --git a/schemas/styles/csl-repository.rnc b/schemas/styles/csl-repository.rnc
index 5a961df..97132be 100644
--- a/schemas/styles/csl-repository.rnc
+++ b/schemas/styles/csl-repository.rnc
@@ -102,7 +102,9 @@ include "csl.rnc" {
       
       ## Select the term to use for et-al abbreviation.
       [ a:defaultValue = "et-al" ]
-      attribute term { "et-al" | "and others" }?,
+      # zotero-chinese/styles 自定义的术语
+      # 国标 10.2.2 节要求姓氏与“et al.”“等”之间留适当空隙
+      attribute term { "et-al" | "and others" | "space-et-al" | "zh-et-al" | "en-et-al" }?,
       font-formatting
     }
 }
diff --git a/schemas/styles/csl-terms.rnc b/schemas/styles/csl-terms.rnc
index 93f3fc5..1f82978 100644
--- a/schemas/styles/csl-terms.rnc
+++ b/schemas/styles/csl-terms.rnc
@@ -8,6 +8,19 @@ div {
     | terms.gender-variants
     | terms.locator
     | item-types
+
+    # 顺序编码制引注中的连接符
+    # 未在 CSL 和 CSL-M 规范中定义，但存在于 citeproc-js 中
+    # https://github.com/zotero-chinese/styles/discussions/439
+    | "citation-range-delimiter"
+
+    # zotero-chinese/styles 自定义的术语
+    # 国标 10.2.2 节要求姓氏与“et al.”“等”之间留适当空隙
+    | "space-et-al"
+    # 中英同时输出时需要
+    | "zh-et-al"
+    | "en-et-al"
+
     | 
       ## Contributor roles
       variables.names
diff --git a/schemas/styles/csl-variables.rnc b/schemas/styles/csl-variables.rnc
index fe66d5e..8288639 100644
--- a/schemas/styles/csl-variables.rnc
+++ b/schemas/styles/csl-variables.rnc
@@ -74,6 +74,8 @@ div {
   ## Title variables
   variables.titles =
     "collection-title"
+    # 中文 style 仓库规定此变量以实现中英同时输出
+    | "original-container-title"
     | "container-title"
     | "original-title"
     | "part-title"
diff --git a/schemas/styles/csl.rnc b/schemas/styles/csl.rnc
index c768055..7dfbbf6 100644
--- a/schemas/styles/csl.rnc
+++ b/schemas/styles/csl.rnc
@@ -362,7 +362,9 @@ div {
   rendering-element.names =
     element cs:names {
       names.attributes,
-      ((names.name?, names.et-al?) & names.label?),
+      ## Allow cs:institution element as child element of cs:names, add
+      ## name-label-before, name-label-after
+      ((names.name?, names.et-al?) & names.label? & names.institution?),
       names.substitute?
     }
   names.attributes =
